variables.tf

```diff
@@ -14,6 +14,13 @@
  * limitations under the License.
  */
 
+variable "image" {
+  type = string
+
+  description = "Source image for the Bastion. If neither source_image nor source_image_family is specified, defaults to the latest public CentOS image."
```

Comment:
nit: the names are slightly different in this module
source_image_family -> image_family
source_image -> image

Reply:
Also I think this description is slightly wrong here, because we do by default set the image_family and image_project to be passed into the instance template inorder to enable shielded_vm by default
```tf
variable "shielded_vm" {
  type = bool

  description = "Enable shielded VM on the bastion host (recommended)"
  default     = true
}

variable "image_family" {
  type = string

  description = "Source image family for the Bastion."
  default     = "centos-7"
}

variable "image_project" {
  type = string

  description = "Project where the source image for the Bastion comes from"
  default     = "gce-uefi-images"
}
```

Reply:
Thanks for the feedback. Will update the description.